docs/user-guide/geo.md

```diff
@@ -0,0 +1,86 @@
+Geospatial Data
+===============
```

Comment:
Let's switch these "symbols under text"-style headers to just use `#` instead.

So:

```markdown
Geospatial Data
============
```

becomes:

```markdown
# Geospatial Data
```

`====` -> `#`
`-----` -> `##`
etc..